alt-tab-macos/ui/PreferencesPanel.swift

```diff
@@ -104,6 +105,12 @@ class PreferencesPanel: NSPanel, NSWindowDelegate {
         return makeLabelWithProvidedControl(labelText, rawName: rawName, control: input, suffixText: suffixText, suffixUrl: suffixUrl)
     }
 
+    private func makeLabelWithCheckbox(_ labelText: String, rawName: String) -> NSStackView {
+        let checkbox = NSButton.init(checkboxWithTitle: labelText, target: nil, action: nil)
+        setControlValue(checkbox, Preferences.rawValues[rawName]!)
+        return makeLabelWithProvidedControl(nil, rawName: rawName, control: checkbox)
```

Comment:
![image](https://user-images.githubusercontent.com/106195/71642643-d9245980-2cf1-11ea-9080-9fc3be2c5ad6.png)

It looks a bit odd that the checkbox has labels on the right. I understand it's OS standard, but in the context of the preferences panel, I think it may look better to do put the text on the left. Did you try it out? Maybe it looks even more odd? I think the code would be:

```swift
let checkbox = NSButton(checkboxWithTitle: "", target: nil, action: nil)
setControlValue(checkbox, Preferences.rawValues[rawName]!)
return makeLabelWithProvidedControl(labelText, rawName: rawName, control: checkbox)
```

Reply:
![Screenshot 2020-01-02 at 10 14 43](https://user-images.githubusercontent.com/58669/71657486-ba9d7d00-2d48-11ea-9a16-cbcb78d5827a.png)

Sure, we can slide it over to the left. Trivial to change later if needed :)